layouts/shortcodes/release-data.html

```diff
@@ -1,6 +1,6 @@
 {{ range $data := .Site.Data.releases.schedule.schedules }}
-{{- $dataVersion := printf "%.2f" $data.release  -}}
-{{- $dataIdVersion := replace ( printf "%.2f" $data.release ) "." "-" | anchorize -}}
+{{- $dataVersion := $data.release -}}
+{{- $dataIdVersion := replace ( $data.release ) "." "-" | anchorize -}}
 <h3 id="release-v{{ $dataIdVersion }}">{{ $dataVersion }}</h3>
```

Comment:
Why drop the `printf`?

It was providing validation (although there's no comment to make that clear): if the version string isn't major.minor, we'd like to fail the build and alert people.

Reply:
Will revert that in https://github.com/kubernetes/website/pull/45927 and drop if from this PR.